watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -640,21 +790,68 @@ def define_state_vars(self):
                 "temperature": self.temperature,
                 "pressure": self.pressure}
 
-    def assert_electroneutrality(self, tol=None, tee=False):
+    def assert_electroneutrality(self, tol=None, tee=True, defined_state=True, adjust_by_ion=None,
+                                 get_property=None, solve=True):
         if tol is None:
-            tol = 1e-6
+            tol = 0
```

Comment:
Is 0 the right default? I worry we'll just be compounding solver error. Maybe `1e-08` or something similar (scaled by the constraint) would be more appropriate?

Reply:
I agree.